omaha/base/constants.h

```diff
@@ -281,6 +281,14 @@ const TCHAR* const kRegValueAutoUpdateJitterMs      = _T("AutoUpdateJitterMs");
 const TCHAR* const kRegValueProxyHost               = _T("ProxyHost");
 const TCHAR* const kRegValueProxyPort               = _T("ProxyPort");
 const TCHAR* const kRegValueMID                     = _T("mid");
+const TCHAR* const kRegValueAlwaysVerifyAuthenticodeSignatures =
+    _T("AlwaysVerifyAuthenticodeSignatures");
+
+// If AlwaysVerifyAuthenticodeSignatures is non-zero in registry (see above),
+// then update binaries with the following extensions are Authenticode-verified:
+const TCHAR* const kAuthenticodeSignedExtensions =
```

Comment:
I suggest defining an array of such extensions, I will comment more on this topic on the code that does the look up.

Reply:
Fixed in bb826fe14b76fe1cf731175a928974a2a97dbbe2.